socketio/src/client/builder.rs

```diff
@@ -36,6 +36,7 @@ pub struct ClientBuilder {
     tls_config: Option<TlsConnector>,
     opening_headers: Option<HeaderMap>,
     transport_type: TransportType,
+    auth: Option<String>,
```

Comment:
I think it might be better to use `serve_json::Value` here instead of string. In that case we're sure that the provided string is valid Json.